sdk/python/setup.py

```diff
@@ -32,6 +32,7 @@
     'cloudpickle',
     'kfp-server-api >= 0.1.18, < 0.1.19', #Update the upper version whenever a new version of the kfp-server-api package is released. Update the lower version when there is a breaking change in kfp-server-api.
     'argo-models == 2.2.1a',  #2.2.1a is equivalent to argo 2.2.1
+    'jsonschema >= 3.0.1'
```

Comment:
Let's also add it to `requirements.txt`

Reply:
Fine with me. But, I thought we are not using the requirements file.

Reply:
If you import something in SDK file (at the top), but not in requirements.txt, then documentation will become empty as `import kfp ` will start failing without the package.

P.S. Ideally we should have only one list of packages.